### ISSUE TITLE
FileStore for initiator resetting sequence number

### ISSUE THREAD

**Issue by lkcole** [quickfixgo/quickfix#406]
I am using FileStoreFactory for managing 1 session where the ResetOn* configuration settings are unset (they all default to N).  

Upon receiving a message from the acceptor to set my sequence number, I stopped the application. After I was sure the process was down, I modified the senderseqnums file associated with the session to the one specified by the acceptor and save. I verified the file was correct and of the appropriate permissions (640). Upon restarting the application, the sequence number populated by the engine was 1 AGAIN. The senderseqnums reflects the same thing.

I've walked through the FileStore.go logic as well as the session.go logic and can't figure out why it would reset. And if it does reset, there should be an event generated.

This appears to be a bug or something glaringly obvious that I'm missing. The pertinent  log is below. Any help in resolving this issue would be greatly appreciated.
```
<2020-06-08 22:41:11.505777213 +0000 UTC, FIX.4.2:XXXXXX/TRADE_CAPTURE/US,CA->CME/68, event>
(Created session)
<2020-06-08 22:41:11.506123792 +0000 UTC, FIX.4.2:XXXXXX/TRADE_CAPTURE/US,CA, event>
(Connecting to: aaa.zzz.yyy.xxx:123456)
<2020-06-08 22:41:11.507635196 +0000 UTC, FIX.4.2:XXXXXX/TRADE_CAPTURE/US,CA->CME/68, event>
(Sending logon request)
MsgSeqNum:  1
2020/06/08 22:41:11 Attempting login with session:  FIX.4.2:XXXXXX/TRADE_CAPTURE/US,CA->CME/68
<2020-06-08 22:41:11.509113112 +0000 UTC, FIX.4.2:XXXXXX/TRADE_CAPTURE/US,CA->CME/68, outgoing>
 (8=FIX.4.29=25935=A34=149=XXXXXX50=TRADE_CAPTURE52=20200608-22:41:11.50756=CME57=68142=US,CA369=0108=30141=N354=20355=Rw9D0zl1pWcKprxUo9tL1400=CME-1-SHA-2561401=431402=f_MugoSdJ-LzAtbUfp4-7bJ42V-m3dFma7G4vjCL52M1603=CME Trade Capture1604=0.11605=Company10=071)
<2020-06-08 22:41:11.512042318 +0000 UTC, FIX.4.2:XXXXXX/TRADE_CAPTURE/US,CA->CME/68, incoming>
(8=FIX.4.29=19635=534=2158369=92052=20200608-22:41:11.51049=CME50=6856=XXXXXX57=TRADE_CAPTURE143=US,CA58=Sequence number received lower than expected. Expected [921] Received 1. Logout forced.789=92110=229)
cme-trade-capture_1  | 2020/06/08 22:41:11 Logged out:  FIX.4.2:XXXXXX/TRADE_CAPTURE/US,CA->CME/68
```

**Comment by estuetzel** [comment]
@lkcole did you figure this out? I am running into the same issue.

**Comment by adam-talos** [comment]
@lkcole @estuetzel You may be running into issue #464

**Comment by lkcole** [comment]
thanks @adam-talos

**Comment by ackleymi** [comment]
Related, the branch for the #464 was merged, see #463 released with version v0.8.1, https://github.com/quickfixgo/quickfix/releases/tag/v0.8.1, please open another issue and link the issue appropriately if it persists
